src/impl_2d.rs

```diff
@@ -27,6 +33,13 @@ where
     /// Return a mutable array view of row `index`.
     ///
     /// **Panics** if `index` is out of bounds.
+    ///
+    /// ```
+    /// # use ndarray::{Array2,array};
+    /// let mut array:Array2<f32> = array![[1f32,2f32],[3f32,4f32]];
+    /// array.row_mut(0)[1] = 5f32;
+    /// assert_eq!(array,array![[1f32,5f32],[3f32,4f32]]);
+    /// ```
```

Comment:
Thanks. Can we use this code style, so that it looks nice and readable? Applies to all the examples This style of floats is what we use throughout the other examples too. Other than that, we want to show the `use` lines and use the conventional spacing around operators and commas.

```rust
    /// use ndarray::{Array2, array};
    /// let mut array: Array2<f32> = array![[1., 2.], [3., 4.]];
    /// array.row_mut(0)[1] = 5.;
    /// assert_eq!(array, array![[1., 5.],[3., 4.]]);
    /// ```
```

Reply:
Putting the `use`s (usually hide them just to minimize examples, but really just whatever library standard is) back in and changing the style of floats is fine.
About renaming `array`, is there a standard name used in other examples?

Reply:
Maybe it's not so important. Array seems to be popular in our examples, just like "a" and "arr" too.